src/transformers/models/auto/configuration_auto.py

```diff
@@ -984,6 +984,23 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
         >>> unused_kwargs
         {'foo': False}
         ```"""
+        token = kwargs.pop("token", None)
```

Comment:
Similar to the `from_pretrained` in `src/transformers/configuration_utils.py`, we want to enable the use of `token`.

(see `_set_token_in_kwargs` in that file)

We keep `use_auth_token` internally however (as the whole `kwargs` are used afterward - and I haven't checked where require to only use `use_auth_token` yet.